lib/stages.ts

```diff
@@ -265,6 +310,16 @@ export class LisaServeApplicationStage extends Stage {
             });
         }
 
+        if (config.securityGroupConfig) {
+            stacks.forEach((lisaStack) => {
+                if (config.securityGroupConfig?.modelSgId) {
```

Comment:
Based on modelSgId being required if securityGroupConfig is defined, I think this check isn't needed